client/maisie/__init__.py

```diff
@@ -53,3 +53,4 @@
 from maisie.resources.projects import Projects
 from maisie.resources.workspaces import Workspaces
 from maisie.resources.users import Users
+from maisie.utils.display import Display
```

Comment:
Adding the import statement to this file will make the `Display` class available to all users of our Maisie package, alongside the `Projects`, `Workspaces`, `Users` classes designed to manage our Maisie instance. 

`Display` is just a helper class that is supposed to be used internally for formatting the responses of other methods, making it explicitly available to users should be discouraged.